cherry_picker/cherry_picker.py

```diff
@@ -753,23 +753,21 @@ def validate_sha(sha):
         )
 
 
-def version_from_branch(branch):
+def version_sort_key(branch):
     """
-    return version information from a git branch name
+    return sort key based on version information from a git branch name
```

Comment:
Can you add more information about how the branches get sorted, perhaps summarize the logic.

Reply:
I wasn't entirely sure how to describe it clearly but I did my best, let me know if it needs further improvements.